datarepo-clienttests/src/main/java/testscripts/BulkLoadScale.java

```diff
@@ -0,0 +1,56 @@
+package testscripts;
+
+import bio.terra.datarepo.api.RepositoryApi;
+import bio.terra.datarepo.client.ApiClient;
+import bio.terra.datarepo.model.*;
+import java.util.List;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import utils.BulkLoadUtils;
+import utils.KubernetesClientUtils;
+
+public class BulkLoadScale extends runner.TestScript {
```

Comment:
This is really minor, but perhaps rename this class to just BulkLoad? I don't think there's anything in here that does any scaling -- that's all in the test configs.